internal/integrationtest/debug/testdata/hardware/my/samd/boards.txt

```diff
@@ -44,3 +45,37 @@ my.debug.cortex-debug.custom.overrideRestartCommands.2=thb setup
 my.debug.cortex-debug.custom.overrideRestartCommands.3=c
 my.debug.cortex-debug.custom.anotherStringParamer=hellooo
 my.debug.svd_file=svd-file
+
+my2.name=My Cool Board
+my2.vid.0=0x2341
+my2.pid.0=0x804e
+my2.upload_port.0.vid=0x2341
+my2.upload_port.0.pid=0x804e
+my2.upload.tool=bossac
+my2.upload.tool.default=bossac
+my2.upload.tool.network=arduino_ota
+my2.upload.protocol=sam-ba
+my2.upload.maximum_size=262144
+my2.upload.maximum_data_size=32768
+my2.upload.use_1200bps_touch=true
+my2.upload.wait_for_upload_port=true
+my2.upload.native_usb=true
+my2.build.mcu=cortex-m0plus
+my2.build.f_cpu=48000000L
+my2.build.usb_product="Arduino MKR1000"
+my2.build.usb_manufacturer="Arduino LLC"
+my2.build.board=SAMD_MY
+my2.build.core=arduino:arduino
+my2.build.extra_flags=-DUSE_ARDUINO_MKR_PIN_LAYOUT -D__SAMD21G18A__ {build.usb_flags}
+my2.build.ldscript=linker_scripts/gcc/flash_with_bootloader.ld
+my2.build.openocdscript=openocd_scripts/arduino_zero.cfg
+my2.build.variant=arduino:mkr1000
+my2.build.vid=0x2341
+my2.build.pid=0x804e
+
+my2.debug.toolchain.path=gcc-path
+my2.debug.toolchain.prefix=gcc-prefix
+my2.debug.server.openocd.path=openocd-path
+my2.debug.server.openocd.scripts_dir=openocd-scripts-dir
+my2.debug.server.openocd.script=single-script
```

Comment:
```suggestion
my2.debug.server.openocd.script=single-script
my2.debug.server.openocd.scripts.0=new-script
```
I would also add a test to see what happens when `scripts.N` is used, probably not in `my2` board but in a new one

Reply:
The latest integration test already checks this (the combined programmer+board+platform configuration has both `script` and `scripts`)